diagram.go

```diff
@@ -7,9 +7,9 @@ import (
 	"github.com/golang/freetype"
 	"github.com/golang/freetype/truetype"
 
-	"github.com/akavel/ditaa/embd"
-	"github.com/akavel/ditaa/fontmeasure"
-	"github.com/akavel/ditaa/graphical"
+	"./embd"
+	"./fontmeasure"
+	"./graphical"
```

Comment:
One would usually do:

    go get -v github.com/akavel/ditaa

and it would automatically clone to proper subpath in $GOPATH (`-v` is optional, just to show what exactly is being downloaded). The only downside is that it registers the "readonly" path as origin remote in git; but `git remote add --push ...` or any other similar stuff works as usual.